src/components/MemoryItem.js

```diff
@@ -10,7 +10,7 @@ MemoryItem.propTypes = {
 export default function MemoryItem({ image, subtitle, onDetail }) {
   return (
     <Wrapper onClick={() => onDetail(image)}>
-      <Image src={image} alt="" width="100%" />
+      <Image src={image} alt="" />
```

Comment:
You should set width and height attributes to give the browser the possibility to calculate the aspect ratio. Beware: don't set any units, but just the values like that: e.g. `<Image src={image} alt="" width="200" height="100" />`